x/concentrated-liquidity/position.go

```diff
@@ -57,6 +57,18 @@ func (k Keeper) initOrUpdatePosition(
 	// TODO: consider deleting position if liquidity becomes zero
 
 	k.setPosition(ctx, poolId, owner, lowerTick, upperTick, position)
+
+	// Update pool's liquidity tree to reflect new position
+	if liquidityDelta.GT(sdk.ZeroDec()) {
```

Comment:
nit: we can store `sdk.ZeroDec` as a global variable to avoid underlying reallocations every time

Reply:
Is there an `IsPositive` helper? It might be helpful here